### PR TITLE
[GEOS-8285] Add clause in GeoServer LICENSE for OSHI library

### PR DESCRIPTION
**Pull request by nmco**
Associated issue: https://osgeo-org.atlassian.net/browse/GEOS-8285

### CODE REVIEW

**Comment by bencaradocdavies** [comment]
@nmco that looks good to me. (But I am not a lawyer.)

**Comment by nmco** [comment]
Thanks for the feedback @bencaradocdavies, any else as any objections on this or it can be merged ?

File: LICENSE.txt [hunk 1]
```diff
@@ -11,21 +11,21 @@ GeoServer is distributed under the GNU General Public License Version 2.0 licens
...
     your distribution. "GeoServer" means the GeoServer software licensed
     under version 2 or any later version of the GPL, or a work based on such
-    software and licensed under the GPL. "Eclipse Libraries" means Eclipse
-    Modeling Framework Project and XML Schema Definition software
-    distributed by the Eclipse Foundation and licensed under the Eclipse
-    Public License Version 1.0 ("EPL"), or a work based on such software and
-    licensed under the EPL.
+    software and licensed under the GPL. "EMF, XSD and OSHI Libraries" means 
+    Eclipse Modeling Framework Project and XML Schema Definition software
+    distributed by the Eclipse Foundation and the OSHI library, all licensed
```
Comment (jodygarnett) [inline LICENSE.txt]: This seems fine, who distributes the OSHI Library?

Seems to be a nice small project - http://oshi.github.io/oshi/team-list.html

Comment (nmco) [reply LICENSE.txt]: I could not find any clear distributor for this project and the copyright holder is the Oshi project team:
```
Copyright (c) 2010 - 2017 The Oshi Project Team
```

Is it required to add an explicit distributor ?

**Comment by jodygarnett** [comment]
it is not required, I checked also.

--
Jody Garnett

**Comment by nmco** [comment]
Thanks for the help @jodygarnett !
OK so this is good to merge ? If nobody complain I'm merging this :smile:

**Pull request merged**